tests/functional/TestSmokeTest/vagrant/roles/gluster/tasks/main.yml

```diff
@@ -6,3 +6,11 @@
 
 - name: start glusterd
   service: name=glusterd state=started enabled=yes
+
+- name: install gluster block
+  yum: name={{ item }} state=present
+  with_items:
+    - gluster-block
```

Comment:
I suggest merging this with the previous block that installs the other gluster packages.